keyboards/melgeek/mj61/readme.md

```diff
@@ -0,0 +1,16 @@
+# MelGeek MJ61
+
+![MJ61](https://drive.google.com/file/d/1siZCVPbHR-L7UVPa6R_-WPJDPoMtis9Q/view)
```

Comment:
limited access.

May want to host this on imgur or the like.

Reply:
> limited access.
> 
> May want to host this on imgur or the like.

Done. Check it please